index.html

```diff
@@ -2,12 +2,75 @@
 <html>
 <head>
 <link href="style.css" rel="stylesheet" type="text/css">
+<script src="https://kit.fontawesome.com/173db1d6d6.js" crossorigin="anonymous"></script>
 <title>Portafolio</title>
 </head>
 <body>
 
-<h1>This is a Heading</h1>
-<p>This is a paragraph.</p>
+    
+<div class="wrapper">
+
+    <nav class="toolbar">
+        <ul>
+            <li class="item">Edahí Gutiérrez </li>        
+            <li id="item-right" class="item">&#9776;</li>                    
+        </ul>        
+    </nav>
+
+    <h1>Hey There. <br>
+        I’m Edahí</h1>
+
+    <h2 id="intro">I’am a Software Developer</h2>   
+
+    <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,  don’t hestiate to contact me. </p>
+
+    <div class="social-group">
+        <ul>
+            <li class="social">
+                <a href="http://github.com/edahigure">
+                    <i class="fa fa-github"></i></a>
+            </li>
+            <li class="social">
+                <a href="https://linkedin.com/in/edahigure">
+                    <i class="fa-brands fa-linkedin-in"></i></a>
+            </li>
+            <li class="social">
+                <a href="https://twitter.com/edahigure">
+                    <i class="fa-brands fa-twitter"></i></a>
+            </li>
+    
+            <li class="social">
+                <a href="#">
+                    <i class="fa-brands fa-angellist"></i></a>
+            </li>
+            
+            <li class="social">        
+                <a href="#">
+                    <i class="fa-solid fa-m"></i>
+                </a>
+        
+            </li>
+
+            <li id="image"> <img src="illustration-header-mobile-cut.png"> </li>
```

Comment:
- Please set this image as a background image instead of placing it as an image.  If an image is not part of the content (aesthetic reasons) use a CSS background image. You can read this [article](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image), for a better understanding.